website/project/views/node.py

```diff
@@ -927,6 +927,7 @@ def _get_summary(node, auth, rescale_ratio, primary=True, link_id=None, show_pat
         'embargo_end_date': node.embargo_end_date.strftime("%A, %b. %d, %Y") if node.embargo_end_date else False,
         'is_pending_embargo': node.is_pending_embargo,
         'archiving': node.archiving,
+        'archive_tree_in_progress': node.root.archiving,
```

Comment:
Is it possible for the `node.root` archiving job to finish before the `node` archiving job, causing it to be displayed like it's not archiving on that node when, in fact, it is?

Regardless, `summary['archiving']` doesn't appear to be used anywhere other than the affected lines below, so it might be better to set `'archiving': node.archiving or node.root.archiving,` and not even need to alter the template.